src/styles/themes/light.css

```diff
@@ -71,39 +74,74 @@
   --input-border: 1px solid var(--dark-grey);
   --input-sidecar-background: var(--grey);
 
-  /* Buttons */
-  --button-primary-background: var(--green);
-  --button-primary-text-color: var(--white);
-  --button-primary-border: none;
-  --button-primary-hover-background: color-mod(var(--green) shade(15%));
-  --button-primary-hover-text-color: var(--white);
-  --button-primary-hover-border: none;
-  --button-primary-active-background: color-mod(var(--green) shade(25%));
-  --button-primary-active-text-color: var(--white);
-  --button-primary-active-border: none;
-
-  --button-inverted-background: var(--white);
-  --button-inverted-text-color: var(--dark-purple);
-  --button-inverted-border: none;
-  --button-inverted-hover-background: color-mod(var(--white) shade(15%));
-  --button-inverted-hover-text-color: var(--dark-purple);
-  --button-inverted-hover-border: none;
-  --button-inverted-active-background: color-mod(var(--white) shade(25%));
-  --button-inverted-active-text-color: var(--dark-purple);
-  --button-inverted-active-border: none;
-
-  --button-outline-background: transparent;
-  --button-outline-text-color: var(--lighter-purple);
-  --button-outline-border: 1px solid var(--lighter-purple);
-  --button-outline-hover-background: transparent;
-  --button-outline-hover-text-color: var(--light-purple);
-  --button-outline-hover-border: 1px solid var(--light-purple);
-  --button-outline-active-background: transparent;
-  --button-outline-active-text-color: var(--purple);
-  --button-outline-active-border: 1px solid var(--purple);
+  /* Button */
+  /*can use color-mod to make deviations instead of manually enter them - currently this follows the figma project values*/
+  /* - https://www.figma.com/file/zaPBj1kYY7bZ7NTC3HDnkU/Kitsu-Buttons?node-id=0%3A1 */
+  --button-d-purple: #ce9fe9; /* for dark mode*/
+  --safe-purple: #8548a8;
+  --safe-grey: #727272;
```

Comment:
What is meant by "safe" in this context?

Reply:
the "safe" keyword are colours I checked were safe to use against white/dark backgrounds, since I was reusing the colour palette already defined. I didn't want to alter those colours nor add new ones in case they were important. 
Thus I added two new colours deviated slightly for better readability. 

currently they should be the same value across all three themes, and it works well. but if needed extra contrast - one can always alter to liking. the original palette just didn't do it for buttons. especially for something like borderless/outline variants.

Reply:
That makes a lot of sense. I have been noticing a lot of cracks in the current color palette approach and thinking of maybe copying what github does and having a set of numbered shades (1-7) for each color. For now this safe thing works and we can clean it up when we figure out a better palette system 😅